azure/common/apigee-deployment.yml

```diff
@@ -48,6 +48,10 @@ parameters:
     type: object
     displayName:  Key/values for custom jinja templating
     default: []
+  - name: has_monitoring
```

Comment:
I'm being very picky here but I personally would prefer a variable name that sounds a bit more like a config option...  `enable_monitoring`? But if your heart is set on `has_monitoring` then keep it.